front/app/containers/Admin/messaging/texting/PreviewMessage/index.tsx

```diff
@@ -0,0 +1,305 @@
+    z-index: 1;
+    bottom: 0;
+    left: -10px;
+    width: 10px;
+    height: 20px;
+    background: white;
+    border-bottom-right-radius: 10px;
+  }
+`;
+
+const TextMessagePreview = (props: WithRouterProps) => {
+  const [showConfirmationModal, setShowConfirmationModal] = useState(false);
+  const [showDeleteTextModal, setShowDeleteTextModal] = useState(false);
+
+  const { campaignId } = props.params;
+  const campaign = useTextingCampaign(campaignId);
+
+  const confirmSendTextingCampaign = async () => {
+    console.log('disable send button here');
+    try {
+      console.log('implement send BE call here');
+      console.log(
+        'if successful, redirect to the view page for the newly created draft message'
+      );
+    } catch (e) {
+      console.log('fail', e);
+    }
+  };
+
+  const confirmDeleteTextingCampaign = async () => {
+    try {
+      const result = await deleteTextingCampaign(campaignId);
+      console.log('successful delete', result);
+      const url = `/admin/messaging/texting`;
+      clHistory.replace(url);
+    } catch (e) {
+      console.log('fail', e);
+    }
+  };
+
+  // actual error state when campaign not found
+  if (isNilOrError(campaign)) return null;
+
+  const { message, phone_numbers } = campaign.attributes;
+
+  return (
+    <>
+      <HelmetIntl
+        title={{ id: 'test', defaultMessage: 'Preview SMS' }}
+        description={{
+          id: 'test',
+          defaultMessage: 'Text message preview',
+        }}
+      />
+      <TextingHeader
+        headerMessage="Preview SMS message"
+        onClickGoBack={() => {
+          const url = `/admin/messaging/texting/${campaignId}/`;
+          clHistory.replace(url);
+        }}
+        showHorizontalRule
+      >
+        <ButtonContainer>
+          <Button
+            onClick={() => {
+              console.log('go back to the create screen');
+            }}
+            buttonStyle="secondary"
+            size="1"
+            text={'Edit'}
+          />
+          <Button
+            onClick={() => {
+              setShowConfirmationModal(true);
+            }}
+            buttonStyle="primary"
+            size="1"
+            icon="send"
+            iconPos="right"
+            text={'Send'}
+          />
+        </ButtonContainer>
+      </TextingHeader>
+      <div>
+        <InformativeTitle>Sending to:</InformativeTitle>
+        <InformativeContent>{phone_numbers.length} people</InformativeContent>
+      </div>
+      <div>
+        <InformativeTitle>Usage:</InformativeTitle>
+        <InformativeContent>
+          {message.length} Characters (message count here)
+        </InformativeContent>
+      </div>
+
+      <PhoneWrapper>
+        <PhoneContainer aria-hidden>
+          <PhoneBezel />
+          <MessagesContainer>
+            <PhoneMessage>{message}</PhoneMessage>
+          </MessagesContainer>
+        </PhoneContainer>
+        <ScreenReaderOnly>{message}</ScreenReaderOnly>
```

Comment:
This is not going to be reachable since the container is aria-hidden. Also, we don't pay attention to a11y in the admin.